src/ocl/handleocl.cpp

```diff
@@ -417,7 +417,7 @@ Program Handle::LoadProgram(const std::string& program_name,
         miopen::SaveBinary(
             path.string(), this->GetTargetProperties(), program_name, params, is_kernel_str);
 #endif
-        return std::move(p);
+        return p;
```

Comment:
This might create a copy, what warning are we fixing here ?

Reply:
mmm ... I'd think the compiler should automatically optimize out the copy in these cases (NRVO or not), nothing to gain by explicitly inhibit it and force the compiler.